lib/litestack/litesearch/model.rb

```diff
@@ -184,7 +184,7 @@ def search(term)
       self.select(
         "#{table_name}.*"
       ).joins(
-        "INNER JOIN #{index_name} ON #{table_name}.#{@schema.get(:type) == :backed ? "rowid" : "id"} = #{index_name}.rowid AND rank != 0 AND #{index_name} MATCH ", Arel.sql("'#{term}'")
+        "INNER JOIN #{index_name} ON #{table_name}.#{@schema.get(:type) == :backed ? "rowid" : primary_key} = #{index_name}.rowid AND rank != 0 AND #{index_name} MATCH ", Arel.sql("'#{term}'")
```

Comment:
I believe if rowid works, then it would be generally faster to use for the join rather than the primary key

Reply:
Maybe forgot about this one, will check it...

Reply:
If you check the last commit, it's finally using only `rowid`